From: "Barak A. Pearlmutter" <barak+git@pearlmutter.net>
Date: Sat, 20 Nov 2021 21:05:08 +0000
Subject: spelling

---
 help.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/help.cpp b/help.cpp
index 9aad118..68c8b41 100644
--- a/help.cpp
+++ b/help.cpp
@@ -4271,7 +4271,7 @@ void Help::setControlStructures() {
       "member\n"
       "        of a list, or the first character of a word) for which the "
       "value\n"
-      "        produced by evaluating the template with that consituent in "
+      "        produced by evaluating the template with that constituent in "
       "its\n"
       "        slot is TRUE.  If there is no such constituent, the empty list\n"
       "        is output.\n"
@@ -4300,7 +4300,7 @@ void Help::setControlStructures() {
       "SUM.\n"
       "        If the data input has only one constituent (member in a list "
       "or\n"
-      "        character in a word), the output is that consituent.  "
+      "        character in a word), the output is that constituent.  "
       "Otherwise,\n"
       "        the template is first applied with ?1 filled with the "
       "next-to-last\n"
