From: "Barak A. Pearlmutter" <barak+git@pearlmutter.net>
Date: Sun, 6 Feb 2022 15:29:43 +0000
Subject: QLogo-cli

---
 CMakeLists.txt | 6 +++---
 mainwindow.cpp | 2 +-
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3065ada..a161d71 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -38,8 +38,8 @@ else()
   add_definitions(-DLOGOPLATFORM="UNIX")
 endif()
 
-add_executable(logo ${LOGO_SOURCES} ${LOGO_HEADERS})
+add_executable(QLogo-cli ${LOGO_SOURCES} ${LOGO_HEADERS})
 
-target_link_libraries(logo Qt5::Core Qt5::Widgets)
+target_link_libraries(QLogo-cli Qt5::Core Qt5::Widgets)
 
-install(TARGETS QLogo logo RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
+install(TARGETS QLogo QLogo-cli RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
diff --git a/mainwindow.cpp b/mainwindow.cpp
index 950526c..5ac7d48 100644
--- a/mainwindow.cpp
+++ b/mainwindow.cpp
@@ -79,7 +79,7 @@ MainWindow::~MainWindow()
 
 int MainWindow::startLogo()
 {
-  QString command = QCoreApplication::applicationDirPath().append("/logo");
+  QString command = QCoreApplication::applicationDirPath().append("/QLogo-cli");
   QStringList arguments;
   arguments << "--QLogoGUI";
 
